src/browser/external_window_event_adapter.js

```diff
@@ -28,15 +28,17 @@ class ExternalWindowEventAdapter {
 
         let cachedState = null;
 
-        ofEvents.on(`external-window/focus/${uuidname}`, () => {
+        const routeExtWin = type => route.externalWindow(type, uuid, name);
```

Comment:
just for consistency, the use of route in this file should look like the others.